components/form/LabeledSearchSelect.vue

```diff
@@ -0,0 +1,378 @@
+<script>
```

Comment:
There's a super duper amount of C&P from LabeledSelect in LabeledSearchSelect. The former has a `searchable` property, can that be used? If not can the functionality in the later be merged in to the former?